packages/Procurement/src/Services/GoodsReceiptManager.php

```
@@ -137,11 +139,12 @@ public function authorizePayment(string $grnId, string $authorizerId): GoodsRece
     /**
      * Get goods receipt note by ID.
      *
+     * @param string $tenantId
      * @param string $grnId
      * @return GoodsReceiptNoteInterface
      * @throws GoodsReceiptNotFoundException
      */
-    public function getGoodsReceipt(string $grnId): GoodsReceiptNoteInterface
+    public function getGoodsReceipt(string $tenantId, string $grnId): GoodsReceiptNoteInterface
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$tenantId'. (undefined)

(UnusedFormalParameter)